deleteMember.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+ function sanitise_input($data)
+ {
+     $data = trim($data);
+     $data = stripslashes($data);
+     $data = htmlspecialchars($data);
+     return $data;
+ }
+
+ if (isset($_POST["submit"])) {
+
+     if (isset($_POST["member_id"]) && !empty($_POST["member_id"])) {
+         $member_id = $_POST["member_id"];
+         $member_id = sanitise_input($member_id);
+         //echo "<p> Member ID : $member_id</p>";
+         if ((!filter_var($member_id, FILTER_VALIDATE_EMAIL))) {
+             //header("location: memberInfo.php?member_id=invalid"); 
+             exit();
+         }
+         require_once("settings.php"); // DB connection info
+
+         $conn = @mysqli_connect($host, $user, $pwd, $sql_db);
+         // check if connection is successful
+         if (!$conn) {
+             // Display error msg
+             echo "<p>Database connection failure </p>";
+             //header("location: memberInfo.php");
+             exit();
+         } else {
+             // Upon successful connection
+             $sql_table = "Member";
+             $user_query = "DELETE * FROM $sql_table WHERE email = '$member_id'";
```

Comment:
The Query should be, 
DELETE FROM $sql_table WHERE email = '$member_id'";

Reply:
okie dokie!!